src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -748,10 +748,10 @@ private void SearchPackagesAndRefreshUpdateCount(bool useCacheForUpdates)
             .PostOnFailure(nameof(PackageManagerControl), nameof(SearchPackagesAndRefreshUpdateCount));
         }
 
-        // Enable if environment variable RecommendNuGetPackages is set to 1.
+        // Check if user has environment variable of RecommendNuGetPackages set to 1 or is in A/B experiment.
         public bool IsRecommenderFlightEnabled()
         {
-            return _forceRecommender;
+            return _forceRecommender | ExperimentationService.Default.IsCachedFlightEnabled("nugetrecommendpkgs");
```

Comment:
Did you mean to use | instead of || ?
Looks like this should work, but it will also evaluate the right side even if the environment variable is set. Was this your intent?

Reply:
I agree with @birgithi, this should be `||`

Reply:
my mistake. thx.